src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model.mdx

```diff
@@ -75,11 +75,13 @@ To determine what user model you're on, from the [account dropdown](/docs/using-
 
 To determine if you can manage users on the New Relic One user model, see [Manage users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/add-manage-users-groups-roles#where).
 
-### Alert-related impacts [#limitations]
+### Impacts related to user model [#limitations]
 
-Users on the New Relic One user model have the following alerting-related impact:
+Users on the New Relic One user model have the following restrictions:
 
-* Alert notifications: These users won’t show up as options when [choosing users to receive alert notifications](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-control-where-send-alerts#user). Instead, you can [use their email address](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-control-where-send-alerts#email).
+* **Alert notifications.** These users won’t show up as an option when [choosing users to receive alert notifications](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-control-where-send-alerts#user). The result of this is that these users will not get [New Relic mobile app push notifications](/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/alerting-new-relic-mobile-apps/). As a workaround, you can instead use their [email address](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-control-where-send-alerts#email) as the notification channel.
```

Comment:
This link: [email address](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-control-where-send-alerts#email) didn't successfully link to the collapser for me.